implementations/rust/ockam/ockam_command/src/upgrade.rs

```diff
@@ -5,42 +5,58 @@ use ockam_core::env::get_env_with_default;
 use serde::Deserialize;
 use std::env;
 
-#[derive(Deserialize)]
-pub struct UpgradeFile {
-    #[serde(default = "default_upgrade_message")]
-    pub upgrade_message: String,
-    #[serde(default = "default_upgrade_message_macos")]
-    pub upgrade_message_macos: String,
-}
+const RELEASE_TAG_NAME_PREFIX: &str = "ockam_v";
 
-fn default_upgrade_message() -> String {
-    "Check out the latest release at https://github.com/build-trust/ockam/releases".to_string()
+#[derive(Deserialize, Debug)]
+struct Release {
+    tag_name: String,
+    html_url: String,
 }
 
-fn default_upgrade_message_macos() -> String {
-    "Run the following command to upgrade the Ockam Command: 'brew install build-trust/ockam/ockam'"
-        .to_string()
+impl Release {
+    fn version(&self) -> Option<&str> {
+        self.tag_name
+            .split_once(RELEASE_TAG_NAME_PREFIX)
+            .map(|(_, version)| version)
+    }
 }
 
 pub fn check_if_an_upgrade_is_available(global_args: &GlobalArgs) {
```

Comment:
It would be super awesome if you could add a small test to make sure we are able to get the expected information from the API endpoint + we can parse it correctly.